qxf2_scheduler/qxf2_scheduler.py

```diff
@@ -38,7 +50,83 @@ def is_weekend(date):
     date = gcal.process_date_string(date)
     day = date.weekday()
     print('~~~~',day)
-    return True if day==5 or day==6 else False 
+    return True if day==5 or day==6 else False
+
+
+def get_free_slots_in_chunks(free_slots):
+    "Return the free slots in 30 minutes interval"
+    #Appending the 30 minutes slot into list
+    if free_slots == None:
+        print("There are no more free slots available for this user")    
+    else:        
+        chunk_time_interval = []        
+        for i in  range(0,len(free_slots)):
```

Comment:
Why this style of for loop? Couldn't you just do:

for free_slot in free_slots:

Reply:
Done